tlatools/org.lamport.tlatools/src/recomp/RVStrategy.java

```diff
@@ -0,0 +1,51 @@
+package recomp;
+
+import java.util.List;
+
+public class RVStrategy implements Runnable {
+    public static boolean globalIsDone = false;
+
+    private String tla;
+    private String cfg;
+    private String recompType;
+    private String recompFile;
+    private boolean verbose;
+
+    public RVStrategy(String tla, String cfg, String recompType, String recompFile, boolean verbose) throws InterruptedException {
+        this.tla = tla;
+        this.cfg = cfg;
+        this.recompType = recompType;
+        this.recompFile = recompFile;
+        this.verbose = verbose;
+    }
+
+    synchronized static public boolean isThreadDone() {
+        boolean isDone = globalIsDone;
+        globalIsDone = true;
+        return isDone;
+    }
+
+    public static void runRVStrategies(String tla, String cfg, String recompType, String recompFile, boolean verbose) throws InterruptedException {
+//		ArrayList<Thread> threads = new ArrayList<>();
+        RVStrategy sampleStrategy = new RVStrategy(tla, cfg, recompType, recompFile, verbose);
+        Thread thread = new Thread(sampleStrategy);
+        thread.start();
+
+        try {
+            thread.join();
+        } catch (InterruptedException e) {
+            System.out.println("Thread interrupted while waiting on something.");
+        }
+        System.exit(99);
```

Comment:
Please delete this line. You should only be calling System.exit() once (in RecompVerify.java)